### PR TITLE
[4.2] remove navigation in options

### PR DESCRIPTION
**Pull request by brianteeman**
Pull Request for Issue #31495
### The problem
All component options are displayed whenever you select the options for any component. This gives the impresion to some users that they can update all options from this page and just click save once. You can't

Why are the other components even present when I click on Template Options

image

image

### CODE REVIEW

**Comment by brianteeman** [comment]
wip

**Comment by ghost** [comment]
I have tested this item :white_check_mark: successfully on 2de65cd9f7c7015968e426e94bad97374b5451e9<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/36807">issues.joomla.org/tracker/joomla-cms/36807</a>.</sub>

**Comment by richard67** [comment]
Does this PR solve issue #31495 so that issue can be closed?

**Comment by brianteeman** [comment]
> Does this PR solve issue #31495 so that issue can be closed?

Its an option not an ideal solution

File: administrator/components/com_config/tmpl/component/default_navigation.php [hunk 1]
```diff
@@ -1,33 +0,0 @@
...
-
-defined('_JEXEC') or die;
-?>
-<ul class="nav flex-column">
-	<?php if ($this->userIsSuperAdmin) : ?>
-		<li class="nav-header"><?php echo Text::_('COM_CONFIG_SYSTEM'); ?></li>
-		<li class="item"><a href="index.php?option=com_config"><?php echo Text::_('COM_CONFIG_GLOBAL_CONFIGURATION'); ?></a></li>
-		<li class="divider"></li>
-	<?php endif; ?>
-	<li class="nav-header"><?php echo Text::_('COM_CONFIG_COMPONENT_FIELDSET_LABEL'); ?></li>
```
Comment (Quy) [inline administrator/components/com_config/tmpl/component/default_navigation.php]: Please deprecate string.

Comment (brianteeman) [reply administrator/components/com_config/tmpl/component/default_navigation.php]: if it moves beyond an rfc

**Pull request closed without merging**